### PR TITLE
adding scripts for preparing new config version

### PR DESCRIPTION
**Pull request by balopat**
try it out with `bash hack/new_version.sh`, let me know what you think! 

@dgageot

### CODE REVIEW

**Comment by dgageot** [comment]
@balopat Sorry I forgot to take a look. Will do that tomorrow

File: hack/new_version.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+sed pkg/skaffold/schema/${PREV_VERSION}/upgrade.go -e "s;$CURRENT_VERSION;$NEW_VERSION;g" > pkg/skaffold/schema/${CURRENT_VERSION}/upgrade.go
+sed -i pkg/skaffold/schema/${CURRENT_VERSION}/upgrade.go -e "s;$PREV_VERSION;$CURRENT_VERSION;g"
+
+sed -i pkg/skaffold/schema/${PREV_VERSION}/upgrade*.go -e "s;latest;$CURRENT_VERSION;g"
+goimports -w pkg/skaffold/schema/${PREV_VERSION}/upgrade*.go
+
+sed -i pkg/skaffold/schema/latest/config.go -e "s;$CURRENT_VERSION;$NEW_VERSION;g"
+
+find integration -name "skaffold.yaml" | xargs -I xx sed -i xx -e "s;$CURRENT_VERSION;$NEW_VERSION;g"
```
Comment (corneliusweig) [inline hack/new_version.sh]: IMHO find/xargs should always be used with `-print0`/`-0` in scripts.
```
find integration -name "skaffold.yaml" -print0 | xargs -0 -I xx sed -i xx -e "s;$CURRENT_VERSION;$NEW_VERSION;g"
```

Comment (balopat) [reply hack/new_version.sh]: why always? in this case it's definitely a relative path operation and everything resolves nicely.

Comment (balopat) [reply hack/new_version.sh]: ah ignore me - I misunderstood the flag.

Comment (balopat) [reply hack/new_version.sh]: thanks for the recommendation :+1:

**Pull request merged**